### ISSUE TITLE
arc will exit after a connection is established

### ISSUE THREAD

**Issue by simongranger** [appium/ruby_console#80]
When running `arc` from `appium_console >= 2.1.0`, I am getting an unexpected exit after the connection is established. I have no issues running tests in rspec the same version of `appium_lib`.

For this run:
`appium_lib v9.7.2`
`appium_console v2.6.0`

[Appium_Server_logs.txt](https://github.com/appium/ruby_console/files/1382944/Appium_Server_logs.txt)

**Comment by KazuCocoa** [comment]
I tried to run `$ ./bin/arc toml test/appium.txt` in this repository and it works.

This library specific setting is the following.

```ruby
  new_command_timeout = { caps: { newCommandTimeout: 999_999 }.merge(opts[:caps] || {}) }
```
But I have no environment to run with 


```
[HTTP] --> POST /wd/hub/session {"desiredCapabilities":{"newCommandTimeout":999999,"platformName":"ios","platformVersion":"10.3","deviceName":"iPhone Simulator","browserName":"Safari","automationName":"XCUITest","someCapability":"some_capability"},"capabilities":{"firstMatch":[{"platformName":"ios","browserName":"Safari"}]}}
[debug] [MJSONWP] Calling AppiumDriver.createSession() with args: [{"newCommandTimeout":999999,"platformName":"ios","platformVersion":"10.3","deviceName":"iPhone Simulator","browserName":"Safari","automationName":"XCUITest","someCapability":"some_capability"},null,{"firstMatch":[{"platformName":"ios","browserName":"Safari"}]}]
[debug] [BaseDriver] Event 'newSessionRequested' logged at 1507909203339 (00:40:03 GMT+0900 (JST))
[Appium] Creating new XCUITestDriver (v2.54.0) session
[Appium] Capabilities:
[Appium]   newCommandTimeout: 999999
[Appium]   platformName: 'ios'
[Appium]   platformVersion: '10.3'
[Appium]   deviceName: 'iPhone Simulator'
[Appium]   browserName: 'Safari'
[Appium]   automationName: 'XCUITest'
[Appium]   someCapability: 'some_capability'
[BaseDriver] The following capabilities were provided, but are not recognized by appium: someCapability.
[BaseDriver] Session created with session id: e05cfa14-dcc1-4926-92bb-cc8e10276557
[debug] [XCUITest] Xcode version set to '9.0' (tools v9.0.0.0.1.1504363082)
[debug] [XCUITest] iOS SDK Version set to '11.0'
[debug] [BaseDriver] Event 'xcodeDetailsRetrieved' logged at 1507909203586 (00:40:03 GMT+0900 (JST))
[debug] [XCUITest] Changing deviceName from 'iPhone Simulator' to 'iPhone 6'
[iOSSim] Constructing iOS simulator for Xcode version 9.0 with udid 'DED4DBAD-8E5E-4AD6-BDC4-E75CF9AD84D8'
[XCUITest] Determining device to run tests on: udid: 'DED4DBAD-8E5E-4AD6-BDC4-E75CF9AD84D8', real device: false
[XCUITest] Safari test requested
[debug] [BaseDriver] Event 'appConfigured' logged at 1507909204244 (00:40:04 GMT+0900 (JST))
[debug] [BaseDriver] Event 'resetStarted' logged at 1507909204245 (00:40:04 GMT+0900 (JST))
[debug] [iOSSim] Cleaning mobile safari data files
[debug] [iOSSim] Checking whether simulator has been run before
[debug] [iOSSim] Simulator has been run before
[debug] [iOSSim] Checking whether simulator has been run before
[debug] [iOSSim] Simulator has been run before
[debug] [iOSSim] Building bundle path map
[debug] [BaseDriver] Event 'resetComplete' logged at 1507909205668 (00:40:05 GMT+0900 (JST))
[debug] [iOSLog] Starting iOS 10.3 simulator log capture
[debug] [iOSLog] System log path: /Users/kazuaki/Library/Logs/CoreSimulator/DED4DBAD-8E5E-4AD6-BDC4-E75CF9AD84D8/system.log
[debug] [BaseDriver] Event 'logCaptureStarted' logged at 1507909205688 (00:40:05 GMT+0900 (JST))
[XCUITest] Setting up simulator
[debug] [iOS] No reason to set locale
[debug] [iOS] No iOS / app preferences to set
[debug] [iOSSim] Setting common Simulator preferences to {"ConnectHardwareKeyboard":false}
[debug] [iOSSim] Updated shared Simulator preferences at '/Users/kazuaki/Library/Preferences/com.apple.iphonesimulator.plist' with {"ConnectHardwareKeyboard":false}
[debug] [iOSSim] The count of running Simulator UI client instances is 1
[iOSSim] Both Simulator with UDID DED4DBAD-8E5E-4AD6-BDC4-E75CF9AD84D8 and the UI client are currently running
[debug] [BaseDriver] Event 'simStarted' logged at 1507909206409 (00:40:06 GMT+0900 (JST))
[XCUITest] Using WDA path: '/Users/kazuaki/GitHub/appium/node_modules/appium-xcuitest-driver/WebDriverAgent'
[XCUITest] Using WDA agent: '/Users/kazuaki/GitHub/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgent.xcodeproj'
[debug] [XCUITest] Capability 'useNewWDA' set to false, so trying to reuse currently running WDA instance at 'http://localhost:8100/'
[debug] [JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[debug] [JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"state\" : \"success\",\n    \"os\" : {\n      \"name\" : \"iOS\",\n      \"version\" : \"10.3.1\"\n    },\n    \"ios\" : {\n      \"simulatorVersion\" : \"10.3.1\",\n      \"ip\" : \"172.254.99.35\"\n    },\n    \"build\" : {\n      \"time\" : \"Oct 12 2017 01:07:55\"\n    }\n  },\n  \"sessionId\" : \"10709AC6-E546-4532-AEBE-2D1EFC40D4E5\",\n  \"status\" : 0\n}"
[debug] [XCUITest] Detected WDA listening at 'http://localhost:8100/'. Setting WDA endpoint to 'http://localhost:8100/'
[debug] [BaseDriver] Event 'wdaStartAttempted' logged at 1507909206463 (00:40:06 GMT+0900 (JST))
[XCUITest] Using provided WebdriverAgent at 'http://localhost:8100/'
[debug] [BaseDriver] Event 'wdaSessionAttempted' logged at 1507909206463 (00:40:06 GMT+0900 (JST))
[debug] [XCUITest] Sending createSession command to WDA
[debug] [JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[debug] [JSONWP Proxy] Got response with status 200: "{\n  \"value\" : {\n    \"state\" : \"success\",\n    \"os\" : {\n      \"name\" : \"iOS\",\n      \"version\" : \"10.3.1\"\n    },\n    \"ios\" : {\n      \"simulatorVersion\" : \"10.3.1\",\n      \"ip\" : \"172.254.99.35\"\n    },\n    \"build\" : {\n      \"time\" : \"Oct 12 2017 01:07:55\"\n    }\n  },\n  \"sessionId\" : \"10709AC6-E546-4532-AEBE-2D1EFC40D4E5\",\n  \"status\" : 0\n}"
[debug] [JSONWP Proxy] Proxying [POST /session] to [POST http://localhost:8100/session] with body: {"desiredCapabilities":{"bundleId":"com.apple.mobilesafari","arguments":["-u","http://0.0.0.0:4723/welcome"],"shouldWaitForQuiescence":true,"shouldUseTestManagerForVisibilityDetection":false,"maxTypingFrequency":60,"shouldUseSingletonTestManager":true}}
[HTTP] --> GET /welcome {}
[debug] [HTTP] Sending welcome response with params: {"message":"Let's browse!"}
[HTTP] <-- GET /welcome 200 10 ms - 124
[debug] [JSONWP Proxy] Got response with status 200: {"value":{"sessionId":"083C8299-A581-46D6-8AFF-2B9D31DB1B14","capabilities":{"device":"iphone","browserName":"Safari","sdkVersion":"10.3.1","CFBundleIdentifier":"com.apple.mobilesafari"}},"sessionId":"083C8299-A581-46D6-8AFF-2B9D31DB1B14","status":0}
[debug] [BaseDriver] Event 'wdaSessionStarted' logged at 1507909209425 (00:40:09 GMT+0900 (JST))
[debug] [BaseDriver] Event 'wdaStarted' logged at 1507909209425 (00:40:09 GMT+0900 (JST))
[debug] [XCUITest] Setting initial orientation to 'PORTRAIT'
[debug] [JSONWP Proxy] Proxying [POST /orientation] to [POST http://localhost:8100/session/083C8299-A581-46D6-8AFF-2B9D31DB1B14/orientation] with body: {"orientation":"PORTRAIT"}
[debug] [JSONWP Proxy] Got response with status 200: {"value":{},"sessionId":"083C8299-A581-46D6-8AFF-2B9D31DB1B14","status":0}
[debug] [BaseDriver] Event 'orientationSet' logged at 1507909210593 (00:40:10 GMT+0900 (JST))
[debug] [XCUITest] Waiting for initial webview
[debug] [iOS] Navigating to most recently opened webview
[debug] [iOS] Retrieving contexts and views
[debug] [iOS] Selecting by url: true (expected url: 'http://0.0.0.0:4723/welcome')
[debug] [RemoteDebugger] useNewSafari --> false
[debug] [RemoteDebugger] Debugger socket connected to ::1:27753
[debug] [RemoteDebugger] Sending connection key request
[debug] [RemoteDebugger] Sending '_rpc_reportIdentifier:' message to remote debugger
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRSimulatorBuildKey":"14E8301","WIRSimulatorProductVersionKey":"10.3.1","WIRSimulatorNameKey":"iPhone"},"__selector":"_rpc_reportSetup:"}'
[debug] [RemoteDebugger] Received response from socket send: '["iPhone","14E8301","10.3.1"]'
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRApplicationDictionaryKey":{"PID:9433":{"WIRRemoteAutomationEnabledKey":false,"WIRIsApplicationActiveKey":1,"WIRIsApplicationReadyKey":false,"WIRApplicationIdentifierKey":"PID:9433","WIRIsApplicationProxyKey":false,"WIRApplicationNameKey":"Safari","WIRApplicationBundleIdentifierKey":"com.apple.mobilesafari"}}},"__selector":"_rpc_reportConnectedApplicationList:"}'
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRDriverDictionaryKey":{}},"__selector":"_rpc_reportConnectedDriverList:"}'
[debug] [RemoteDebugger] Notified of connected drivers: {}.
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRRemoteAutomationEnabledKey":false,"WIRIsApplicationActiveKey":1,"WIRIsApplicationReadyKey":false,"WIRApplicationIdentifierKey":"PID:9438","WIRIsApplicationProxyKey":true,"WIRApplicationNameKey":"com.apple.WebKit.WebContent","WIRApplicationBundleIdentifierKey":"com.apple.WebKit.WebContent","WIRHostApplicationIdentifierKey":"PID:9433"},"__selector":"_rpc_applicationConnected:"}'
[debug] [RemoteDebugger] Notified that new application 'PID:9438' has connected
[debug] [RemoteDebugger] Current applications available:
[debug] [RemoteDebugger] Application: 'PID:9433'
[debug] [RemoteDebugger]     id: "PID:9433"
[debug] [RemoteDebugger]     isProxy: false
[debug] [RemoteDebugger]     name: "Safari"
[debug] [RemoteDebugger]     bundleId: "com.apple.mobilesafari"
[debug] [RemoteDebugger]     hostId: undefined
[debug] [RemoteDebugger]     isActive: 1
[debug] [RemoteDebugger]     isAutomationEnabled: false
[debug] [RemoteDebugger] Application: 'PID:9438'
[debug] [RemoteDebugger]     id: "PID:9438"
[debug] [RemoteDebugger]     isProxy: true
[debug] [RemoteDebugger]     name: "com.apple.WebKit.WebContent"
[debug] [RemoteDebugger]     bundleId: "com.apple.WebKit.WebContent"
[debug] [RemoteDebugger]     hostId: "PID:9433"
[debug] [RemoteDebugger]     isActive: 1
[debug] [RemoteDebugger]     isAutomationEnabled: false
[debug] [RemoteDebugger]     pageDict: "Waiting for data"
[debug] [RemoteDebugger] Found app id key 'PID:9433' for bundle 'com.apple.mobilesafari'
[debug] [RemoteDebugger] Found separate bundleId 'com.apple.WebKit.WebContent' acting as proxy for 'com.apple.mobilesafari', with app id 'PID:9438'
[debug] [RemoteDebugger] Using proxied app id 'PID:9438'
[debug] [RemoteDebugger] Connected to application
[debug] [RemoteDebugger] Sim name: iPhone
[debug] [RemoteDebugger] Sim build: 14E8301
[debug] [RemoteDebugger] Sim platform version: 10.3.1
[debug] [RemoteDebugger] Selecting application
[debug] [RemoteDebugger] Found app id key 'PID:9433' for bundle 'com.apple.mobilesafari'
[debug] [RemoteDebugger] Found separate bundleId 'com.apple.WebKit.WebContent' acting as proxy for 'com.apple.mobilesafari', with app id 'PID:9438'
[debug] [RemoteDebugger] Trying out the possible app ids: PID:9438
[debug] [RemoteDebugger] Selecting app PID:9438 (try #1 of 20)
[debug] [RemoteDebugger] Sending '_rpc_forwardGetListing:' message to remote debugger
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRApplicationIdentifierKey":"PID:9438","WIRListingKey":{"1":{"WIRTitleKey":"Appium/welcome","WIRTypeKey":"WIRTypeWeb","WIRURLKey":"http://0.0.0.0:4723/welcome","WIRPageIdentifierKey":1}}},"__selector":"_rpc_applicationSentListing:"}'
[debug] [RemoteDebugger] Received response from socket send: '["PID:9438",{"1":{"WIRTitleKey":"Appium/welcome...'
[debug] [RemoteDebugger] Page changed: {"1":{"WIRTitleKey":"Appium/welcome","WIRTypeKey":"WIRTypeWeb","WIRURLKey":"http://0.0.0.0:4723/welcome","WIRPageIdentifierKey":1}}
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRRemoteAutomationEnabledKey":false,"WIRIsApplicationActiveKey":1,"WIRIsApplicationReadyKey":true,"WIRApplicationIdentifierKey":"PID:9438","WIRIsApplicationProxyKey":true,"WIRApplicationNameKey":"com.apple.WebKit.WebContent","WIRApplicationBundleIdentifierKey":"com.apple.WebKit.WebContent","WIRHostApplicationIdentifierKey":"PID:9433"},"__selector":"_rpc_applicationUpdated:"}'
[debug] [RemoteDebugger] Notified that application 'PID:9438' has been updated.
[debug] [RemoteDebugger] Current applications available:
[debug] [RemoteDebugger] Application: 'PID:9433'
[debug] [RemoteDebugger]     id: "PID:9433"
[debug] [RemoteDebugger]     isProxy: false
[debug] [RemoteDebugger]     name: "Safari"
[debug] [RemoteDebugger]     bundleId: "com.apple.mobilesafari"
[debug] [RemoteDebugger]     hostId: undefined
[debug] [RemoteDebugger]     isActive: 1
[debug] [RemoteDebugger]     isAutomationEnabled: false
[debug] [RemoteDebugger] Application: 'PID:9438'
[debug] [RemoteDebugger]     id: "PID:9438"
[debug] [RemoteDebugger]     isProxy: true
[debug] [RemoteDebugger]     name: "com.apple.WebKit.WebContent"
[debug] [RemoteDebugger]     bundleId: "com.apple.WebKit.WebContent"
[debug] [RemoteDebugger]     hostId: "PID:9433"
[debug] [RemoteDebugger]     isActive: 1
[debug] [RemoteDebugger]     isAutomationEnabled: false
[debug] [RemoteDebugger]     pageDict: [{"id":1,"title":"Appium/welcome","url":"http://0.0.0.0:4723/welcome","isKey":false}]
[debug] [RemoteDebugger] Finally selecting app PID:9438: [{"id":1,"title":"Appium/welcome","url":"http://0.0.0.0:4723/welcome","isKey":false}]
[debug] [iOS] Picking webview 'WEBVIEW_9438.1'
[debug] [iOS] Attempting to set context to 'WEBVIEW_9438.1'
[debug] [RemoteDebugger] Selecting page '1' on app 'PID:9438' and forwarding socket setup
[debug] [RemoteDebugger] Sending '_rpc_forwardSocketSetup:' message to remote debugger
[debug] [RemoteDebugger] Sender key set
[debug] [RemoteDebugger] Sending '_rpc_forwardSocketData:' message to remote debugger
[debug] [RemoteDebugger] Received applicationSentData response
[debug] [RemoteDebugger] Found data handler for response
[debug] [RemoteDebugger] Received data response from socket send: '{}'
[debug] [RemoteDebugger] Original command: enablePage
[debug] [RemoteDebugger] Enabled activity on page
[debug] [RemoteDebugger] Checking document readyState
[debug] [RemoteDebugger] Sending javascript command (function (){ return document.readyState; })()
[debug] [RemoteDebugger] Sending '_rpc_forwardSocketData:' message to remote debugger
[debug] [RemoteDebugger] Received applicationSentData response
[debug] [RemoteDebugger] Found data handler for response
[debug] [RemoteDebugger] Received data response from socket send: 'complete'
[debug] [RemoteDebugger] Original command: sendJSCommand
[debug] [RemoteDebugger] readyState was "complete"
[debug] [RemoteDebugger] Unregistering from page readiness notifications
[debug] [BaseDriver] Event 'initialWebviewNavigated' logged at 1507909210743 (00:40:10 GMT+0900 (JST))
[Appium] New XCUITestDriver session created successfully, session e05cfa14-dcc1-4926-92bb-cc8e10276557 added to master session list
[debug] [BaseDriver] Event 'newSessionStarted' logged at 1507909210744 (00:40:10 GMT+0900 (JST))
[debug] [MJSONWP] Responding to client with driver.createSession() result: {"webStorageEnabled":false,"locationContextEnabled":false,"browserName":"Safari","platform":"MAC","javascriptEnabled":true,"databaseEnabled":false,"takesScreenshot":true,"networkConnectionEnabled":false,"newCommandTimeout":999999,"platformName":"ios","platformVersion":"10.3","deviceName":"iPhone Simulator","automationName":"XCUITest","someCapability":"some_capability","udid":"DED4DBAD-8E5E-4AD6-BDC4-E75CF9AD84D8"}
[HTTP] <-- POST /wd/hub/session 200 7416 ms - 489
[HTTP] --> GET /wd/hub/status {}
[debug] [MJSONWP] Calling AppiumDriver.getStatus() with args: []
[debug] [MJSONWP] Responding to client with driver.getStatus() result: {"build":{"version":"1.7.2-beta","revision":"ee872e6362ec3d5620c3bc3f7f176861df99b7c1"}}
[HTTP] <-- GET /wd/hub/status 200 18 ms - 126
[HTTP] --> POST /wd/hub/session/e05cfa14-dcc1-4926-92bb-cc8e10276557/timeouts/implicit_wait {"ms":30000}
[debug] [MJSONWP] Calling AppiumDriver.implicitWait() with args: [30000,"e05cfa14-dcc1-4926-92bb-cc8e10276557"]
[debug] [XCUITest] Executing command 'implicitWait'
[debug] [BaseDriver] Set implicit wait to 30000ms
[debug] [MJSONWP] Responding to client with driver.implicitWait() result: null
[HTTP] <-- POST /wd/hub/session/e05cfa14-dcc1-4926-92bb-cc8e10276557/timeouts/implicit_wait 200 5 ms - 76
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRApplicationIdentifierKey":"PID:9433","WIRListingKey":{}},"__selector":"_rpc_applicationSentListing:"}'
[debug] [RemoteDebugger] Received page change notice for app 'PID:9433' but listening for 'PID:9438'. Ignoring.
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRRemoteAutomationEnabledKey":false,"WIRIsApplicationActiveKey":1,"WIRIsApplicationReadyKey":true,"WIRApplicationIdentifierKey":"PID:9433","WIRIsApplicationProxyKey":false,"WIRApplicationNameKey":"Safari","WIRApplicationBundleIdentifierKey":"com.apple.mobilesafari"},"__selector":"_rpc_applicationUpdated:"}'
[debug] [RemoteDebugger] Notified that application 'PID:9433' has been updated.
[debug] [RemoteDebugger] Current applications available:
[debug] [RemoteDebugger] Application: 'PID:9433'
[debug] [RemoteDebugger]     id: "PID:9433"
[debug] [RemoteDebugger]     isProxy: false
[debug] [RemoteDebugger]     name: "Safari"
[debug] [RemoteDebugger]     bundleId: "com.apple.mobilesafari"
[debug] [RemoteDebugger]     hostId: undefined
[debug] [RemoteDebugger]     isActive: 1
[debug] [RemoteDebugger]     isAutomationEnabled: false
[debug] [RemoteDebugger]     pageDict: []
[debug] [RemoteDebugger] Application: 'PID:9438'
[debug] [RemoteDebugger]     id: "PID:9438"
[debug] [RemoteDebugger]     isProxy: true
[debug] [RemoteDebugger]     name: "com.apple.WebKit.WebContent"
[debug] [RemoteDebugger]     bundleId: "com.apple.WebKit.WebContent"
[debug] [RemoteDebugger]     hostId: "PID:9433"
[debug] [RemoteDebugger]     isActive: 1
[debug] [RemoteDebugger]     isAutomationEnabled: false
[debug] [RemoteDebugger]     pageDict: [{"id":1,"title":"Appium/welcome","url":"http://0.0.0.0:4723/welcome","isKey":false}]
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRApplicationIdentifierKey":"PID:9438","WIRListingKey":{"1":{"WIRTitleKey":"Appium/welcome","WIRTypeKey":"WIRTypeWeb","WIRURLKey":"http://0.0.0.0:4723/welcome","WIRPageIdentifierKey":1}}},"__selector":"_rpc_applicationSentListing:"}'
[debug] [RemoteDebugger] Page changed: {"1":{"WIRTitleKey":"Appium/welcome","WIRTypeKey":"WIRTypeWeb","WIRURLKey":"http://0.0.0.0:4723/welcome","WIRPageIdentifierKey":1}}
[debug] [iOS] Remote debugger notified us of a new page listing: {"appIdKey":"9438","pageArray":[{"id":1,"title":"Appium/welcome","url":"http://0.0.0.0:4723/welcome","isKey":false}]}
[debug] [iOS] No key id found. Choosing first id from page array
[debug] [iOS] Checking if page needs to load
[debug] [iOS] New page listing is same as old, doing nothing
[debug] [RemoteDebugger] Receiving data from remote debugger: '{"__argument":{"WIRApplicationIdentifierKey":"PID:9438","WIRListingKey":{"1":{"WIRTitleKey":"Appium/welcome","WIRTypeKey":"WIRTypeWeb","WIRURLKey":"http://0.0.0.0:4723/welcome","WIRPageIdentifierKey":1}}},"__selector":"_rpc_applicationSentListing:"}'
[debug] [RemoteDebugger] Page changed: {"1":{"WIRTitleKey":"Appium/welcome","WIRTypeKey":"WIRTypeWeb","WIRURLKey":"http://0.0.0.0:4723/welcome","WIRPageIdentifierKey":1}}
[debug] [iOS] Remote debugger notified us of a new page listing: {"appIdKey":"9438","pageArray":[{"id":1,"title":"Appium/welcome","url":"http://0.0.0.0:4723/welcome","isKey":false}]}
[debug] [iOS] No key id found. Choosing first id from page array
[debug] [iOS] Checking if page needs to load
[debug] [iOS] New page listing is same as old, doing nothing
```


I found the following in your log.

```
[HTTP] --> POST /wd/hub/session/60173aeb-c752-488b-a436-6a4686514dbd/timeouts/implicit_wait {"ms":0}
```

In this case, new command timeout is zero.

Could you set the following capability as a **String**? I guess current setting is _Integer_.
```
[Appium]   newCommandTimeout: 60
```

@simongranger

**Comment by simongranger** [comment]
I tried setting it as a string, but I still get the same issue.
Please see in attached logs a succesfull (rpsec) vs unsucessful (arc) connection
[Disconnected.txt](https://github.com/appium/ruby_console/files/1383216/Disconnected.txt)
[NoDisconnect.txt](https://github.com/appium/ruby_console/files/1383217/NoDisconnect.txt)

**Comment by KazuCocoa** [comment]
hm...I compared the two files.

We can see `[YouiEngineDriver] Connection closed` in disconnected text after `[debug] [XCUITest] Reset requested. Removing app with id 'tv.youi.ticketbus' from the device1`

**Comment by simongranger** [comment]
but later you see it connect: 
[YouiEngineDriver] Attempt #1
[YouiEngineDriver] Connecting to WebDriver: 169.254.109.226:12345
[YouiEngineDriver] Connected
Let me grab logs of a run with `appium_console v2.0.1`

**Comment by simongranger** [comment]
[success2.0.1.txt](https://github.com/appium/ruby_console/files/1383430/success2.0.1.txt)

**Comment by simongranger** [comment]
The logs look very similar to me.
Is there a way to debug arc itself?

**Comment by x3au** [comment]
I am also seeing the exact same issue. Arc exits after the app is launched in simulator.

**Comment by KazuCocoa** [comment]
I could reproduce it. Let me check...

**Comment by KazuCocoa** [comment]
I guess it happens when you use `pry 0.11.x` as pry command.
For quick fix, please use `pry 0.10.4` which is the latest pry as `0.10.x`
@simongranger @x3au

**Comment by KazuCocoa** [comment]
I've released the new version which requires pry `0.11.x` and fix the starting pry issue for `0.11.x`.
